server/controllers/userController.js

```diff
@@ -113,94 +123,143 @@ const UserController = {
         }
       ]
     };
+    query.attributes = userDetails.user;
+    query.limit = request.query.limit || null;
+    query.offset = request.query.offset || null;
+    query.order = [['createdAt', 'DESC']];
     db.users
-      .findAll(query)
+      .findAll({ query: query, limit: query.limit, offset: query.offset })
```

Comment:
To avoid this error flag from houndci use this shorthand
`.findAll({ query, limit: query.limit, offset: query.offset })`